book/src/build/cargo.md

```diff
@@ -144,6 +144,49 @@ manifest key][links]* in the Cargo reference.
 
 [links]: https://doc.rust-lang.org/cargo/reference/build-scripts.html#the-links-manifest-key
 
+## Features
+
+CXX provides features to enable additional functionality such as compiling
+the CXX lib with a different standard.
+
+Features are enabled on the CXX crate for Cargo-based setups, and with compiler
+options for non-Cargo setups.
+
+The following features are available, see `[features]` section in [Cargo.toml](https://github.com/dtolnay/cxx/blob/master/Cargo.toml) for a full list:
+<table>
+<tr><th>feature</th><th>compiler option</th><th>description</th></tr>
+<tr><td>c++14</td><td>-std=c++14 (gcc, clang), /std:c++14 (msvc)</td><td>Compile CXX lib with c++14</td></tr>
+<tr><td>c++17</td><td>-std=c++17 (gcc, clang), /std:c++17 (msvc)</td><td>Compile CXX lib with c++17</td></tr>
+</table>
+
+## Additional C++ standard features
```

Comment:
Potential documentation for enabling additional C++ APIs.

Nicer to look at page:
https://github.com/roligugus/cxx/blob/string_view/book/src/build/cargo.md#additional-c-standard-features